test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -6691,33 +6691,33 @@ BOOST_AUTO_TEST_CASE(storage_array_ref)
 		contract Store is BinarySearch {
 			uint[] data;
 			function add(uint v) public {
-				data.push(0);
+				data.push();
```

Comment:
Removed the argument because of https://github.com/ethereum/solidity/issues/7840